packages/react-aria-components/src/FileTrigger.tsx

```diff
@@ -52,7 +52,14 @@ function FileTrigger(props: FileTriggerProps, ref: ForwardedRef<HTMLDivElement>)
       <PressResponder onPress={() => inputRef.current?.click()}>
         {children}
       </PressResponder>
-      <Input type="file" style={{display: 'none'}} ref={inputRef} accept={acceptedFileTypes?.toString()} onChange={onChange} capture={defaultCamera} multiple={allowsMultiple} /> 
+      <Input 
+        type="file" 
+        style={{display: 'none'}} 
+        ref={inputRef} 
+        accept={acceptedFileTypes?.toString()} 
+        onChange={(e: ChangeEvent<HTMLInputElement>) => e.target.files && onChange(e.target.files)} 
```

Comment:
I think this can be shortened to this:

```suggestion
        onChange={(e) => onChange?.(e.target.files)} 
```

and the default `onChange` of `= () => {}` can be removed if so.